### PR TITLE
chore(cts): assert the retry strategy error

### PR DESCRIPTION
**Pull request by millotp**
## 🧭 What and Why

Suggested in #3334, we can add tests for the retry strategy error message, even if its different for every language.

### CODE REVIEW

**Review by morganleroi — APPROVED** [review]
Nice ! Thanks (a lot) for doing it.

File: tests/CTS/client/search/api.json [hunk 1]
```diff
@@ -87,6 +87,47 @@
...
+        }
+      },
+      {
+        "type": "method",
+        "method": "customGet",
+        "parameters": {
+          "path": "1/test/hang/${{language}}"
+        },
+        "expected": {
+          "error": {
```
Comment (shortcuts) [inline tests/CTS/client/search/api.json:114]: if we add a new language, will this throw? I did not saw this specific case

Comment (millotp) [reply tests/CTS/client/search/api.json:114]: yes the CTS will add a default message and the test will fail, so its necessary to add it here

**Review by shortcuts — APPROVED** [review]
gg!!!!!!!!

**Pull request merged**